ISICkonto/Flow/AppFlowController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  AppFlowController.swift
+//  ISICkonto
+//
+//  Created by Vendula Švastalová on 14/02/2019.
+//  Copyright © 2019 Vendula Švastalová. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import RxCocoa
+import RxSwift
+
+protocol AppFlowControllerProtocol {
+    var navigationController: UINavigationController { get }
+}
+
+class AppFlowController : AppFlowControllerProtocol {
+    
+    var navigationController: UINavigationController
```

Comment:
❔ má nějaký důvod, proč to není `let`?

Reply:
kdyz jsem tu classu vytvarela tak me napadlo, ze ten flow controller by ve vetsi appce mohl mit vice flows pod sebou a kazdy flow zacina svym navigation controllerem, tak jedine tak, ale kdyz nad tim tak premyslim tak si opet kazdy to flow drzi svuj NavigationController, jen mezi nema prechazis, takze spis asi `let`